channel-email/src/main/java/com/synopsys/integration/alert/channel/email/convert/EmailGlobalConfigurationModelSaveActions.java

```diff
@@ -48,22 +47,16 @@ public void updateConcreteModel(ConfigurationModel configurationModel) {
         Optional<UUID> defaultConfigurationId = configurationAccessor.getConfiguration()
             .map(EmailGlobalConfigModel::getId)
             .map(UUID::fromString);
-        Optional<EmailGlobalConfigModel> emailGlobalConfigModel = emailFieldModelConverter.convert(configurationModel);
+        Optional<EmailGlobalConfigModel> emailGlobalConfigModel = emailFieldModelConverter.convertAndValidate(configurationModel);
         if (defaultConfigurationId.isPresent() && emailGlobalConfigModel.isPresent()) {
```

Comment:
The emailGlobalConfigModel.isPresent() isn't needed.  The ifPresent is used on the next line.  Remove the isPresent() call on the email model.
```suggestion
        if (defaultConfigurationId.isPresent()) {
```